src/CMakeLists.txt

```diff
@@ -297,6 +297,14 @@ if(DNNL_BLAS_VENDOR STREQUAL "ACCELERATE")
     DESTINATION ${LIB_CONFIG_INSTALL_DIR})
 endif()
 
+if((DNNL_GPU_RUNTIME STREQUAL "DPCPP" OR DNNL_GPU_RUNTIME STREQUAL "SYCL") AND DNNL_GPU_VENDOR STREQUAL "NVIDIA")
+    install(FILES
+        "../cmake/FindcuBLAS.cmake"
+        "../cmake/FindcublasLt.cmake"
+        "../cmake/FindcuDNN.cmake"
+        DESTINATION ${LIB_CONFIG_INSTALL_DIR})
+endif()
```

Comment:
Do I get it right that you are adding those cmake files to the install target?
Also why in `src/CMakeLists.txt` and not in `cmake/SYCL.cmake` ?

Reply:
> Do I get it right that you are adding those cmake files to the install target?

Yes, with this change those files will get installed as they are used by the `find_package(...)` calls in the generated `dnnl-config.cmake` file in `install_dir/lib/cmake`.

> Also why in src/CMakeLists.txt and not in cmake/SYCL.cmake ?

I saw that a few other files are installed here and thought it would be good to keep all similar calls in one place. I am happy to move them to `SYCL.cmake` if that's preferable?